src/modules/musig/session_impl.h

```diff
@@ -180,17 +192,69 @@ int secp256k1_musig_pubnonce_parse(const secp256k1_context* ctx, secp256k1_musig
             return 0;
         }
     }
-    /* The group elements can not be infinity because they were just parsed */
     secp256k1_musig_pubnonce_save(nonce, ge);
     return 1;
 }
 
+/* Outputs 33 zero bytes if the given group element is the point at infinity and
+ * otherwise outputs the compressed serialization */
+void secp256k1_ge_serialize_ext(unsigned char *out33, secp256k1_ge* ge) {
```

Comment:
```suggestion
static void secp256k1_ge_serialize_ext(unsigned char *out33, secp256k1_ge* ge) {
```
Well I think this should be done for every function not exposed in the public API but maybe it's better to postpone this to a PR that tidies the entire codebase... I suspect it's wrong in other modules too.

Reply:
I think that plus another function were the only offenders in the module. Fixed.